index.html

```diff
@@ -14,19 +14,36 @@
 </head>
 
 <body>
+
     <header>
         <a href="#">Kemboi</a>
         <nav>
             <img class="header-icon" src="images/union.svg" />
             <ul>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
                 <img class="header-mail" src="images/mail.svg" alt="Email" />
             </ul>
         </nav>
     </header>
 
+    <div class="navMenu" id="navMenu">
+        <button class="menuBtn">
+            <img src="./images/cross.svg" alt="">
+        </button>
+        <ul>
+            <li>
+                <a href="#portfolio">Portfolio</a>
+                <hr>
+            </li>
+            <li><a href="#about">About</a></li>
+            <hr>
+            <li><a href="#contact">Contact</a></li>
+            <hr>
+        </ul>
+    </div>
+
```

Comment:
👏 Excellent job on implementing the navigation items using an unordered list! It's a great addition to your project. I do have a couple of recommendations to further improve your code:

- [ ] Instead of using a `div` tag to wrap the container of your navigation items, I suggest using a `nav` tag. This will enhance the semantic structure of your HTML and improve accessibility. It's an important practice to follow for creating more meaningful and accessible web pages.

- [ ] Additionally, considering the significance of the mobile menu in the header section, it's a good practice to place the navigation inside the header tag. This way, you can ensure that all header-related elements are grouped together, improving the overall organization of your HTML.

By making these updates, your code will not only meet best practices but also enhance the clarity and structure of your project. Keep up the great work, and I'm confident your attention to detail will make your project shine! 🌟📝🚀